src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.component.ts

```diff
@@ -129,6 +161,22 @@ export class CodeEditorAceComponent implements AfterViewInit, OnChanges, OnDestr
             // Reset the undo stack after file change, otherwise the user can undo back to the old file
             this.editor.getEditor().getSession().setUndoManager(new ace.UndoManager());
             this.displayAnnotations();
+
+            // Setup inline feedbacks
+            // Get amount of lines of code in order to render for each line a corresponding inline feedback component
+            if (this.isTutorAssessment) {
+                const lines = this.editor.getEditor().getSession().getLength();
+                this.lineCounter = new Array(lines);
+                if (!this.feedbacks) {
+                    this.feedbacks = [];
+                }
+                this.fileFeedbacks = this.feedbacks.filter((feedback) => feedback.reference && feedback.reference.includes(this.selectedFile));
+                this.fileFeedbackPerLine = {};
+                this.fileFeedbacks.forEach((feedback) => {
+                    const line: number = +feedback.reference!.split('line:')[1];
```

Comment:
is the + in `+feedback.reference!.split('line:')[1];` a typo?

Reply:
Nope it is not, with that you transform a `string` to a `number` :)